Update lgcalibrationstudio.sh

macmangr opened the pull request:
appNewVersion cannot determine the correct version from the existing URL. The URL adjusted

2024-12-01 16:53:31 : REQ   : lgcalibrationstudio : ################## Start Installomator v. 10.6, date 2024-12-01
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## Version: 10.6
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## Date: 2024-12-01
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## lgcalibrationstudio
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : DEBUG mode 1 enabled.
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : setting variable from argument DEBUG=0
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : name=LG Calibration Studio
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : appName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : type=pkgInZip
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : archiveName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : downloadURL=https://gscs-b2c.lge.com/downloadFile?fileId=glgdWtFQRpBh9q23mGFQ
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : curlOptions=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : appNewVersion=7.2.5
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : appCustomVersion function: Not defined
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : versionKey=CFBundleShortVersionString
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : packageID=LGSI.TrueColorPro
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : pkgName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : choiceChangesXML=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : expectedTeamID=5SKT5H4CPQ
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : blockingProcesses=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : installerTool=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : CLIInstaller=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : CLIArguments=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateTool=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateToolArguments=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateToolRunAsCurrentUser=
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : BLOCKING_PROCESS_ACTION=tell_user
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : NOTIFY=success
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : LOGGING=DEBUG
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Label type: pkgInZip
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : archiveName: LG Calibration Studio.zip
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : no blocking processes defined, using LG Calibration Studio as default
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Changing directory to /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : No version found using packageID LGSI.TrueColorPro
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : App(s) found: /Applications/LG Calibration Studio.app
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : found app at /Applications/LG Calibration Studio.app, version 7.2.5, on versionKey CFBundleShortVersionString
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : appversion: 7.2.5
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Latest version of LG Calibration Studio is 7.2.5
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : There is no newer version available.
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Deleting /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Debugging enabled, Deleting tmpDir output was:
/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Installomator did not close any apps, so no need to reopen any apps.
2024-12-01 16:53:32 : REQ   : lgcalibrationstudio : No newer version.
2024-12-01 16:53:32 : REQ   : lgcalibrationstudio : ################## End Installomator, exit code 0

macmangr committed:
Update lgcalibrationstudio.sh

appNewVersion cannot determine the correct version from the existing URL. The URL adjusted

2024-12-01 16:53:31 : REQ   : lgcalibrationstudio : ################## Start Installomator v. 10.6, date 2024-12-01
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## Version: 10.6
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## Date: 2024-12-01
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : ################## lgcalibrationstudio
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : DEBUG mode 1 enabled.
2024-12-01 16:53:31 : INFO  : lgcalibrationstudio : setting variable from argument DEBUG=0
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : name=LG Calibration Studio
2024-12-01 16:53:31 : DEBUG : lgcalibrationstudio : appName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : type=pkgInZip
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : archiveName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : downloadURL=https://gscs-b2c.lge.com/downloadFile?fileId=glgdWtFQRpBh9q23mGFQ
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : curlOptions=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : appNewVersion=7.2.5
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : appCustomVersion function: Not defined
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : versionKey=CFBundleShortVersionString
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : packageID=LGSI.TrueColorPro
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : pkgName=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : choiceChangesXML=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : expectedTeamID=5SKT5H4CPQ
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : blockingProcesses=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : installerTool=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : CLIInstaller=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : CLIArguments=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateTool=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateToolArguments=
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : updateToolRunAsCurrentUser=
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : BLOCKING_PROCESS_ACTION=tell_user
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : NOTIFY=success
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : LOGGING=DEBUG
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Label type: pkgInZip
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : archiveName: LG Calibration Studio.zip
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : no blocking processes defined, using LG Calibration Studio as default
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Changing directory to /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : No version found using packageID LGSI.TrueColorPro
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : App(s) found: /Applications/LG Calibration Studio.app
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : found app at /Applications/LG Calibration Studio.app, version 7.2.5, on versionKey CFBundleShortVersionString
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : appversion: 7.2.5
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Latest version of LG Calibration Studio is 7.2.5
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : There is no newer version available.
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Deleting /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : DEBUG : lgcalibrationstudio : Debugging enabled, Deleting tmpDir output was:
/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.7qrxzTumcN
2024-12-01 16:53:32 : INFO  : lgcalibrationstudio : Installomator did not close any apps, so no need to reopen any apps.
2024-12-01 16:53:32 : REQ   : lgcalibrationstudio : No newer version.
2024-12-01 16:53:32 : REQ   : lgcalibrationstudio : ################## End Installomator, exit code 0

bartreardon commented:
🤖 Validation robot 🤖
File fragments/labels/lgcalibrationstudio.sh
** Label info:
├ ✅ correct line ending
├ ✅ Name: LG Calibration Studio
├  ✅ Type: pkgInZip
├  ✅ Expected Team: 5SKT5H4CPQ
├  ✅ App New Version: 7.2.9
└ Download URL: https://www.lg.com/us/lg-partner-store" title="opens in a new window" target="_blank" aria-label="Partner Employees, this link opens in a new window" data-testid="MenuItem-10" menu-type="MenuItem" menu-index="10" tabindex="0"><span class="MuiTypography-root MuiTypography-body2 css-1yfgukv">Partner Employees<!-- --> <svg class="MuiSvgIcon-root MuiSvgIcon-colorAction MuiSvgIcon-fontSizeSmall css-146ak1n" focusable="false" aria-hidden="true" viewBox="0 0 24 24" data-testid="OpenInNewOutlinedIcon"><path d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3z"></path></svg></span></a></div></li></ul><ul class="MuiList-root MuiList-padding css-12dzmx4" tabindex="-1
  └ ❌ URL is not reachable - error 0
** WARNING: Some checks failed
✅ 5 checks passed
❌ 1 checks failed
❌ Please review [Contributing to Installomator](https://github.com/Installomator/Installomator/wiki/Contributing-to-Installomator) and update your pull request.
****

macmangr committed:
Update lgcalibrationstudio.sh

Adjusted the label to correctly retrieve the latest version and download link of the software.

2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : setting variable from argument DEBUG=0
2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : Total items in argumentsArray: 1
2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : argumentsArray: DEBUG=0
2025-07-20 15:19:19 : REQ   : lgcalibrationstudio : ################## Start Installomator v. 10.8, date 2025-07-20
2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : ################## Version: 10.8
2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : ################## Date: 2025-07-20
2025-07-20 15:19:19 : INFO  : lgcalibrationstudio : ################## lgcalibrationstudio
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : Reading arguments again: DEBUG=0
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : BLOCKING_PROCESS_ACTION=tell_user
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : NOTIFY=success
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : LOGGING=INFO
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : Label type: pkgInZip
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : archiveName: LG Calibration Studio.zip
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : no blocking processes defined, using LG Calibration Studio as default
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : No version found using packageID LGSI.TrueColorPro
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : App(s) found: /Applications/LG Calibration Studio.app
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : found app at /Applications/LG Calibration Studio.app, version 7.2.9, on versionKey CFBundleShortVersionString
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : appversion: 7.2.9
2025-07-20 15:19:20 : INFO  : lgcalibrationstudio : Latest version of LG Calibration Studio is 7.4.2
2025-07-20 15:19:20 : REQ   : lgcalibrationstudio : Downloading https://gscs-b2c.lge.com/downloadFile?fileId=4SwM5SGNKulUluThvngkg to LG Calibration Studio.zip
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : Downloaded LG Calibration Studio.zip – Type is  Zip archive data, at least v2.0 to extract, compression method=deflate – SHA is 209660ca94fa44782cac272be6a562eb39d481b6 – Size is 82276 kB
2025-07-20 15:19:23 : REQ   : lgcalibrationstudio : no more blocking processes, continue with update
2025-07-20 15:19:23 : REQ   : lgcalibrationstudio : Installing LG Calibration Studio
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : Unzipping LG Calibration Studio.zip
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : found pkg: /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.c6gmzExt62/LGCalibrationStudio_V7.4.2_signed.pkg
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : Verifying: /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.c6gmzExt62/LGCalibrationStudio_V7.4.2_signed.pkg
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : Team ID: 5SKT5H4CPQ (expected: 5SKT5H4CPQ )
2025-07-20 15:19:23 : INFO  : lgcalibrationstudio : Checking package version.
2025-07-20 15:19:24 : INFO  : lgcalibrationstudio : Downloaded package LGSI.TrueColorPro version
2025-07-20 15:19:24 : INFO  : lgcalibrationstudio : Installing /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.c6gmzExt62/LGCalibrationStudio_V7.4.2_signed.pkg to /
2025-07-20 15:19:41 : INFO  : lgcalibrationstudio : Finishing...
2025-07-20 15:19:44 : INFO  : lgcalibrationstudio : No version found using packageID LGSI.TrueColorPro
2025-07-20 15:19:44 : INFO  : lgcalibrationstudio : App(s) found: /Applications/LG Calibration Studio.app
2025-07-20 15:19:44 : INFO  : lgcalibrationstudio : found app at /Applications/LG Calibration Studio.app, version 7.4.2, on versionKey CFBundleShortVersionString
2025-07-20 15:19:44 : REQ   : lgcalibrationstudio : Installed LG Calibration Studio
2025-07-20 15:19:44 : INFO  : lgcalibrationstudio : notifying
2025-07-20 15:19:45 : INFO  : lgcalibrationstudio : Installomator did not close any apps, so no need to reopen any apps.
2025-07-20 15:19:45 : REQ   : lgcalibrationstudio : All done!
2025-07-20 15:19:45 : REQ   : lgcalibrationstudio : ################## End Installomator, exit code 0

2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : setting variable from argument DEBUG=0
2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : Total items in argumentsArray: 1
2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : argumentsArray: DEBUG=0
2025-07-20 15:19:49 : REQ   : lgcalibrationstudio : ################## Start Installomator v. 10.8, date 2025-07-20
2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : ################## Version: 10.8
2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : ################## Date: 2025-07-20
2025-07-20 15:19:49 : INFO  : lgcalibrationstudio : ################## lgcalibrationstudio
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : Reading arguments again: DEBUG=0
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : BLOCKING_PROCESS_ACTION=tell_user
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : NOTIFY=success
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : LOGGING=INFO
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : Label type: pkgInZip
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : archiveName: LG Calibration Studio.zip
2025-07-20 15:19:50 : INFO  : lgcalibrationstudio : no blocking processes defined, using LG Calibration Studio as default
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : No version found using packageID LGSI.TrueColorPro
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : App(s) found: /Applications/LG Calibration Studio.app
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : found app at /Applications/LG Calibration Studio.app, version 7.4.2, on versionKey CFBundleShortVersionString
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : appversion: 7.4.2
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : Latest version of LG Calibration Studio is 7.4.2
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : There is no newer version available.
2025-07-20 15:19:51 : INFO  : lgcalibrationstudio : Installomator did not close any apps, so no need to reopen any apps.
2025-07-20 15:19:51 : REQ   : lgcalibrationstudio : No newer version.
2025-07-20 15:19:51 : REQ   : lgcalibrationstudio : ################## End Installomator, exit code 0

gilburns reviewed (changes requested):
@macmangr - The packageID line in unnecessary. The standard version detection can work for this app. Can you update this PR to remove that line? Everything else seems to check out and work.

macmangr committed:
Update lgcalibrationstudio.sh

Removed packageID line

uurazzle reviewed (approved):
Hi @macmangr,

Thanks for the updates on this Installomator label. I am approving this PR, but please review and address any remaining precheck/lint notes listed above where applicable.

gilburns reviewed (approved):
Thank you for your contribution to the project!
